django/conf/global_settings.py

```diff
@@ -216,6 +216,11 @@ def gettext_noop(s):
 # Default form rendering class.
 FORM_RENDERER = "django.forms.renderers.DjangoTemplates"
 
+# RemovedInDjango60Warning: It's a transitional setting helpful in early
+# adoption of "https" as the new default value of forms.URLField.assume_scheme.
+# Set True to assume "https" during the Django 5.x release cycle.
+EARLY_ADOPT_URLFIELD_ASSUME_HTTPS = False
```

Comment:
IMO we don't need to include “early adopt” in the name - that’s contextual information about the setting, but it doesn’t describe what the setting does.

```suggestion
URLFIELD_ASSUME_HTTPS = False
```

Perhaps the name could be explicit about being form-related?

```suggestion
FORMS_URLFIELD_ASSUME_HTTPS = False
```

Reply:
Changed to `FORMS_URLFIELD_ASSUME_HTTPS`.